services/vehicle/src/main/java/com/wedriveu/services/vehicle/boundary/vehicleregister/RegisterPublisherVerticle.java

```diff
@@ -0,0 +1,35 @@
+package com.wedriveu.services.vehicle.boundary.vehicleregister;
+
+import com.wedriveu.services.shared.rabbitmq.VerticlePublisher;
+import com.wedriveu.services.shared.utilities.Constants;
+import com.wedriveu.services.vehicle.rabbitmq.Messages;
+import io.vertx.core.Future;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.json.JsonObject;
+
+import static com.wedriveu.services.shared.utilities.Constants.*;
+
+/**
+ *
+ * Vert.x RabbitMQ Publisher for vehicle register response. Replies the vehicle with the adding result to the database.
+ * @author Marco Baldassarri
+ * @since 08/08/2017.
```

Comment:
Cit. "The @since tag should be used to define which version you added the method, class, etc. This is your hint to other developers that they should only expect the method when they run against a particular version of the package."